config/tests/jobs/jobs_test.go

```diff
@@ -156,31 +157,34 @@ func TestReportTemplate(t *testing.T) {
 
 func TestURLTemplate(t *testing.T) {
 	testcases := []struct {
-		name    string
-		jobType kube.ProwJobType
-		org     string
-		repo    string
-		job     string
-		build   string
-		expect  string
+		name       string
+		jobType    kube.ProwJobType
+		org        string
+		repo       string
+		job        string
+		build      string
+		expect     string
+		skipNonK8s bool
```

Comment:
The name of this field is confusing. Is `k8sOnly` an appropriate rename?

Reply:
sure, done